src/System.Data.SqlClient/src/System/Data/SqlClient/TdsEnums.cs

```diff
@@ -571,6 +571,7 @@ public enum FedAuthLibrary : byte
         // Login data validation Rules
         //
         internal const ushort MAXLEN_HOSTNAME = 128; // the client machine name
+        // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Not a password.")]
```

Comment:
should this be on the line below?

Reply:
This one is weird. The 
```
        internal const ushort MAXLEN_USERNAME = 128; // the client user id
        internal const ushort MAXLEN_PASSWORD = 128; // ...
```
is considered as password even with the new line (something like `MAXLEN_USERNAME =  MAXLEN_PASSWORD = 128` ) 
This is why the suppression message is on line which doesn't look like password. Perhaps the justification message can be better.